lib/profile/sections/education.dart

```diff
@@ -10,48 +10,21 @@ class ProfileEducation extends StatelessWidget {
   
   @override
   Widget build(BuildContext context) {
-    return Container(
-      width: MediaQuery.of(context).size.width,
-      padding: EdgeInsets.fromLTRB(30.0, 10.0, 30.0, 20.0),
-      decoration: BoxDecoration(
-        border: Border(
-          bottom: BorderSide(
-            width: 5.0,
-            color: Colors.black12
-          )
-        )
-      ),
-      child: Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: <Widget>[
-          SectionHeader(title: sectionTitleEducation),
-          buildSchoolList(context)
-        ],
-      )
+    return Section(
+      title: sectionTitleEducation,
+      child: buildSchoolList(context)
     );
   }
 
   buildSchoolRow(BuildContext context, String school, String program, String startYear, String endYear) {
     final String dateRange = startYear + " - " + endYear;
 
-    return Container(
-      width: MediaQuery.of(context).size.width,
-      padding: EdgeInsets.fromLTRB(0.0, 10.0, 0.0, 10.0),
-      decoration: BoxDecoration(
-        border: Border(
-          bottom: BorderSide(
-            color: Colors.black12
-          )
-        )
-      ),
-      child: Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: <Widget>[
-          buildEducationText(school, true, false),
-          buildEducationText(program, false, false),
-          buildEducationText(dateRange, false, true)
-        ]
-      )
+    return SectionRow(
+      children: <Widget>[
+        buildEducationText(school, true, false),
```

Comment:
Similar thing here. You could extract this information into a config as well, import it here, and then loop over it for the reasons stated above. 

```dart
// In a file like secttions-config.dart
const educationSectionConfig = {
   school: {
      key1: "value1", 
      key2: "value2"
      key 3: "value3"
    }, 
    ...
}

// In education.dart
for (var detail in educationConfig) {
   var { key1, key2, key3} = detail;
   buildEducationText(key1, key2, key3)
} 
```

If we need to ever make changes in what we render there, it's done through the config and has generalized behaviour.

Reply:
I'm a little confused. Doesn't this only work for constants?

Reply:
In this example, if I created a separate config file, what do I assign `key1` assuming it's replacing the `school` parameter?

Reply:
We could place functions as values in the config. But looking at Nayan's comment below, I think it'd be best if we wait until we use the backend to optimize the retrieval of these values. 

Sorry if this was a conversation that was repeated LOL.

Reply:
Kk sounds good!